apps/saleor-app-checkout/backend/environment.ts

```diff
@@ -7,6 +7,12 @@ const maskToken = (token: string) => "*".repeat(Math.max(token.length - 4, 0)) +
 
 export const getAuthToken = () => {
   let token;
+
+  if (IS_TEST) {
+    // Allows to use real appToken to record requests in development
+    token = "TEST";
+  }
```

Comment:
Why not just use `serverEnvVars.appToken` in tests? Is this condition even necessary?

Reply:
You're right, I've moved the logic of setting env var into `setupTestsBeforeEnv` file. We do the same thing for encryption secret